src/main/java/uk/gov/hmcts/reform/pip/subscription/management/repository/SubscriptionRepository.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.pip.subscription.management.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.pip.subscription.management.models.Subscription;
+
+import java.util.List;
+
+@Repository
+public interface SubscriptionRepository extends JpaRepository<Subscription, Long> {
+    /**
```

Comment:
if you could add the class javadocs above the class and same with method javadocs 😊

Reply:
Yep, sorry - won't happen again!